chapters/install/os/via-pip.rst

```diff
@@ -0,0 +1,346 @@
+
+Install Jina Core via ``pip``
+=================================
+
+``pip`` is a package-management system written in Python used to install and manage software packages. For Python 2.x the command is called ``pip``\ , and for Python >= 3 it is called ``pip3``. In many operating systems ``pip`` is linked to ``pip3``. Run the command ``pip --version`` to check the Python version that is in use, actually.
+
+If you prefer to run Jina natively on your host, please make sure you have Python >= 3.7 installed.
+
+Install from PyPi
+-----------------
+
+On Linux/Mac, simply run:
+
+.. code-block:: bash
+
+   pip install jina
+
+.. note:: Currently we do not support native Python on Windows. If you are a Windows user, one workaround is to `run Jina on Windows Subsystem for Linux <on-wsl.md>`_ or `run Jina in a Docker container <via-docker.md>`_.
+
+Check the installation
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+After you have installed Jina Core, it is advisable to carry out a short check. This will verify that everything has gone smoothly. On the command line, run the following command:
+
+.. code-block:: bash
+
+   jina -vf
```

Comment:
i think calling `jina -f` is sufficiently enough?

Reply:
@nan-wang ?